.github/workflows/test.yml

```diff
@@ -68,7 +68,7 @@ jobs:
         run: |
           docker run --name test-cassandra -d -p 9042:9042 \
           -v ${GITHUB_WORKSPACE}/modevo-script/dat/inp/creationSchema.cql:/creationSchema.cql \
-          cassandra:3.11
+          cassandra:4
```

Comment:
Cassandra uses minor versions to identify LTS. To avoid eventual unexpected upgrades specify the minor version